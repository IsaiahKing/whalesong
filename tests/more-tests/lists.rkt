#lang planet dyoo/whalesong/base

'(1 2 3)
(list "hello" "world")

(empty? empty)
(empty? '())
(cons? '())
(cons? '(hello))
(first '(hello))
(rest '(hello))
(second '(hello world))