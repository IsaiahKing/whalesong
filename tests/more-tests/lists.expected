(1 2 3)
("hello" "world")
true
true
false
true
hello
()
world
